store/postgres/src/entities.rs

```diff
@@ -792,14 +841,50 @@ impl Connection {
         self.conn.batch_execute(&*query)?;
 
         match *GRAPH_STORAGE_SCHEME {
-            v::Relational => Layout::create_relational_schema(
-                &self.conn,
-                &schema_name,
-                schema.id.clone(),
-                &schema.document,
-            )
-            .map(|_| ()),
-            v::Split => create_split_schema(&self.conn, &schema_name),
+            v::Relational => {
+                let layout = Layout::create_relational_schema(
+                    &self.conn,
+                    &schema_name,
+                    schema.id.clone(),
+                    &schema.document,
+                )?;
```

Comment:
All this is happening inside one transaction. Postgres (unlike a lot of other databases) makes DDL transactional, meaning if the transaction gets rolled back because of an error farther down the line, all the DDL statements we issued get rolled back, too.

Reply:
Sorry, I now see this is running inside `econn.transaction(...)`, which of course means everything's rolled back in case of an error. Nevermind 😄